app/controllers/Auth.scala

```diff
@@ -231,6 +248,19 @@ class Auth @Inject() (
     Ok(views.html.auth.startResetPassword(emailForm))
   }
 
+  def handlePool1StartResetPassword(email64: String) = Action.async { implicit request =>
+    val email = new String(java.util.Base64.getDecoder.decode(email64), "UTF-8")
+    userService.retrieve(LoginInfo(CredentialsProvider.ID, email)).flatMap {
+        case None => Future.successful(Redirect(routes.Auth.startResetPassword()).flashing("error" -> Messages("error.noUser")))
+        case Some(user) => for {
+          token <- userTokenService.save(UserToken.create(user.id, email, isSignUp = false))
+        } yield {
+          mailer.resetPassword(email, link = routes.Auth.resetPassword(token.id.toString).absoluteURL())
```

Comment:
`resetPassword`? Does the retrieved email contains a clear explanation why the user got this email? If it explains that the user initiated the reset password workflow, the user will become uncertain and will suspect a bug or a hack...